templates/base.html

```diff
@@ -8,7 +8,19 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
     <title>{% block head_title %}Entérate, Ayuda, Informa - Suministros Puerto Rico{% endblock %}</title>
-
+    <meta name="description" content="La donación de suministros a las áreas más afectadas del país por el terremoto ha sido un éxito. Esta página pretende ser un vehículo de distribución equitativa para estos suministros, administrada por la misma gente que dona, y que recibe."/>
+    <meta property="og:locale" content="es_ES" />
+    <meta property="og:type" content="website" />
+    <meta property="og:title" content="Entérate, Ayuda, Informa - Suministros Puerto Rico" />
+    <meta property="og:url" content="https://suministrospr.com/" />
+    <meta property="og:site_name" content="Suministros Puerto Rico" />
+    <meta property="og:image" content="/img/social_banner.png" />
```

Comment:
@crm114 puedes rebase from master? Añadí un util para absolute static urls.

```suggestion
    <meta property="og:image" content="{% absolute_static 'img/social_banner.png' %}" />
```